cli/cli.go

```diff
@@ -458,6 +461,50 @@ func getCreateInitialMigrationSubcommand() cli.Command {
 	}
 }
 
+func getGenerateGoextCommand() cli.Command {
+	return cli.Command{
+		Name:  "converter",
+		Usage: "Generates code used by golang extensions",
+		Description: `gohan converter [path to file with schemas] [flags...]
+
+Converter generates code from yaml schemas.
+Generated code:
+	* Definition of structs representing objects from each schema
+	* Interfaces for getters and setters for these objects
+	* Implementation of these interfaces by pointers to generated structs
+	* Interfaces that can be extended
+	* Constructors for objects with default values
+	* Database functions for generated structs (fetch, list)
+ARGUMENTS:
+	There is one argument - path to file with yaml schemas
+`,
+		Flags: []cli.Flag{
+			cli.StringFlag{Name: "goext-package", Value: "goext", Usage: "Package name for golang extension interfaces"},
+			cli.StringFlag{Name: "crud-package", Value: "goodies", Usage: "Package name for crud functions"},
+			cli.StringFlag{Name: "raw-package", Value: "resources", Usage: "Package name for raw structs"},
+			cli.StringFlag{Name: "interface-package", Value: "esi", Usage: "Package name for interfaces"},
```

Comment:
Do not expose `esi` to OSS gohan.

Reply:
@morrisson I've changed default interface package name to "interfaces".